plugins/kontent-ai-plugin/package.json

```diff
@@ -8,15 +8,15 @@
     "setup": "node ./scripts/setup.js"
   },
   "dependencies": {
-    "@kontent-ai/gatsby-source": "9.0.1",
-    "@kontent-ai/gatsby-components": "9.0.1",
-    "gatsby-plugin-manifest": "^4.7.0",
-    "uuid": "^8.3.2"
+    "@kontent-ai/gatsby-source": "10.0.0",
+    "@kontent-ai/gatsby-components": "10.0.0",
+    "gatsby-plugin-manifest": "^5.4.0",
+    "uuid": "^9.0.0"
   },
   "devDependencies": {
     "@kontent-ai/backup-manager": "4.0.1",
-    "chalk": "4",
-    "inquirer": "^8.2.0",
-    "yargs-parser": "^21.0.0"
+    "chalk": "5.2.0",
+    "inquirer": "^9.1.4",
+    "yargs-parser": "^21.1.1"
```

Comment:
I'm inclined to say let's leave these dependencies alone, as the same versions are used across all the other flavors of the homepage starter.

Reply:
Yeah — using the latest version of `chalk` is problematic as it's `ES module` now:

<img width="1191" alt="Screen Shot 2023-01-17 at 9 26 55 AM" src="https://user-images.githubusercontent.com/1790506/212924647-08ce554e-ee88-47e9-82b0-8e27f241487a.png">